src/content/en/fundamentals/performance/critical-rendering-path/measure-crp.md

```diff
@@ -5,20 +5,51 @@ description: Learn to measure the critical rendering path.
 {# wf_updated_on: 2014-09-17 #}
 {# wf_published_on: 2014-03-31 #}
 
-# Measuring the Critical Rendering Path with Navigation Timing {: .page-title }
+# Measuring the Critical Rendering Path {: .page-title }
 
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 
-You can't optimize what you can't measure. Thankfully, the Navigation 
-Timing API gives us all the necessary tools to measure each step of the 
-critical rendering path!
+The foundation of every solid performance strategy is good measurement and
+instrumentation. You can't optimize what you can't measure. This doc
+explains different approaches for measuring CRP performance.
 
-### TL;DR {: .hide-from-toc }
-- Navigation Timing provides high resolution timestamps for measuring CRP.
-- Browser emits series of consumable events which capture various stages of the CRP.
+The key difference between the different approaches is the different kind
+of metrics that they can capture.
 
+* The Lighthouse approach runs a series of automated tests against a page,
+  and then generates a report on the page's CRP performance. This approach
+  provides a quick and easy high-level overview of CRP performance, but does
+  not truly reflect how users experience the page.
+* The Navigation Timing API approach captures [Real User
+  Monitoring (RUM)](https://en.wikipedia.org/wiki/Real_user_monitoring)
+  metrics. As the name implies, these metrics are captured from real user
+  interactions with your site. This approach provides a more accurate view
+  of CRP performance, but is more difficult and time-consuming to set up.
```

Comment:
"with your site, which provides an accurate view into the real-world CRP performance, as experienced your users across a variety of devices and network conditions". 

Scratch the but is more difficult..